docs/components_and_bundles/bundles/SyliusCalendarBundle/summary.rst

```diff
@@ -0,0 +1,10 @@
+Summary
+=======
+
+Date handling for PHP applications.
+
+Bug tracking
+------------
+
+This bundle uses `GitHub issues <https://github.com/Sylius/Calendar/issues>`_.
```

Comment:
Should we track here or in Sylius/Sylius ??

Reply:
The code isn't a part of Sylius/Sylius repository, so issues related to this bundle should be tracked on `Sylius/Calendar` repository (IMO).